.github/workflows/release.yml

```
@@ -61,9 +62,9 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
-      - name: Create zip files
+      - name: Create chrome zip file
         if: steps.changesets.outcome == 'success' && steps.changesets.outputs.published == 'true'
-        run: npm run dist chrome && npm run dist firefox
+        run: npm run dist:chrome
```

These need to be separated out since each of them require a full rebuild, which wipes the `.zip` files.

---

Makes sense. Thoughts on keeping the separation between build and dist commands so it's clear we're doing a fresh build here?

```suggestion
        run: npm run build -- --env TARGET=chrome && npm run dist -- chrome
```